### ISSUE TITLE
Tensorflow has not supported operator

### ISSUE THREAD

**Issue by DavideCatto** [Microsoft/MMdnn#127]
Platform: win10:
Python: 3.6
Framework: Tensorflow 1.6 with GPU GTX1080
Destination framework: Mxnet

When I run the script
`python -m mmdnn.conversion._script.convertToIR -f tensorflow -d my_model -n model.ckpt-25.meta -w model.ckpt-25 --dstNodeName Mean` I get these errors/warnings:

`Parse file [model.ckpt-25.meta] with binary format successfully.
Tensorflow model file [model.ckpt-25.meta] loaded successfully.
Tensorflow checkpoint file [model.ckpt-25] loaded successfully. [32] variables loaded.
Tensorflow has not supported operator [Iterator] with name [Iterator].
Tensorflow has not supported operator [NoOp] with name [IteratorGetNext:1].
Tensorflow has not supported operator [NoOp] with name [IteratorGetNext:2].
Tensorflow has not supported operator [IteratorGetNext] with name [IteratorGetNext].
Tensorflow has not supported operator [Pack] with name [netvlad/Slice/begin].
Tensorflow has not supported operator [Pack] with name [netvlad/Slice_1/begin].
Tensorflow has not supported operator [Pack] with name [netvlad/Slice_2/begin].
Tensorflow has not supported operator [Square] with name [netvlad/l2_normalize_3/Square].
Tensorflow has not supported operator [Square] with name [netvlad/l2_normalize_6/Square].
Tensorflow has not supported operator [Square] with name [netvlad/l2_normalize/Square].
Tensorflow has not supported operator [Sum] with name [netvlad/l2_normalize_3/Sum].
Tensorflow has not supported operator [Sum] with name [netvlad/l2_normalize_6/Sum].
Tensorflow has not supported operator [Sum] with name [netvlad/l2_normalize/Sum].
Tensorflow has not supported operator [Maximum] with name [netvlad/l2_normalize_3/Maximum].
Tensorflow has not supported operator [Maximum] with name [netvlad/l2_normalize_6/Maximum].
Tensorflow has not supported operator [Maximum] with name [netvlad/l2_normalize/Maximum].
Tensorflow has not supported operator [Rsqrt] with name [netvlad/l2_normalize_3/Rsqrt].
Tensorflow has not supported operator [Rsqrt] with name [netvlad/l2_normalize_6/Rsqrt].
Tensorflow has not supported operator [Rsqrt] with name [netvlad/l2_normalize/Rsqrt].
Tensorflow has not supported operator [ExpandDims] with name [netvlad/expanddim_1].
Tensorflow has not supported operator [ExpandDims] with name [netvlad/expanddim_2].
Tensorflow has not supported operator [ExpandDims] with name [netvlad/expanddim].
Tensorflow has not supported operator [Shape] with name [netvlad/Shape_2].
Tensorflow has not supported operator [Shape] with name [netvlad/Shape_3].
Tensorflow has not supported operator [Shape] with name [netvlad/Shape_4].
Tensorflow has not supported operator [Shape] with name [netvlad/Shape_5].
Tensorflow has not supported operator [Shape] with name [netvlad/Shape].
Tensorflow has not supported operator [Shape] with name [netvlad/Shape_1].
Tensorflow has not supported operator [Slice] with name [netvlad/Slice_1].
Tensorflow has not supported operator [Slice] with name [netvlad/Slice_2].
Tensorflow has not supported operator [Slice] with name [netvlad/Slice].
Tensorflow has not supported operator [Softmax] with name [netvlad/Softmax_1].`
And so on..

How can I solve these? There are some operators not supported yet or I have commit some errors? I'm converting a Tensorflow model (.meta, .index, .data) not from slim checkpoint; could be the problem?

**Comment by kitstar** [comment]
Hi @DavideCatto, could you share your model for us to reproduce? Thanks.

**Comment by DavideCatto** [comment]
Of course.
[https://drive.google.com/file/d/1HrA1SQCVzLMHgbG3IzZelihMoDVSDbSa/view?usp=sharing](url)
Excluded the first part where there is the "Iterator" (it will be replaced by a Placeholder so no error should compare), the other problems seem obtained by ad L2_normalize layers conversion from Tensorflow to IR intermediate model. Which are the not supported operations? How to solve these errors? 

Add infos: this is a vgg16 model with Netvlad layer add before fc, trained using triplet loss and shared weights.

Thanks for the reply.

**Comment by DavideCatto** [comment]
@namizzz thanks for the reply.
Yes, a lot of errors are caused by the input layer and I have already removed, but the remaning unsupported layer is the L2_normalize and the expand_dimension layer.

**Comment by ToBigboss** [comment]
> @namizzz thanks for the reply.
> Yes, a lot of errors are caused by the input layer and I have already removed, but the remaning unsupported layer are the L2_normalize and the expand_dimension layer.

Hi @DavideCatto , Is the problem solved?  how you deal with it?
